### DISCUSSION TITLE
My Vercel site re-builds upon repo update, but isn't getting updated

### DISCUSSION THREAD

**Discussion by AvivShvitzky** [vercel/community#778] [Help] [answered]
Hello,
My Vercel site re-builds when I push code to my github repo, it shows no errors, but when I log into the site, it shows the content of an old branch (just a former version of my main branch. I only have a main branch).
It looks like the site did not update for a couple of repo updates, even though it does do a process of rebuilding.
I tried to redeploy but to no avail.

Any tips?

**Comment by amyegan** [comment, marked as answer]
Hi @AvivShvitzky! If you share the deployment ID or URL, I can look into the logs and try to see if anything is wrong there. 

It would also be helpful if you can share your repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) so I can try to see the problem.

**Reply by AvivShvitzky** [reply]
IDK why I only saw a successful build on the main page, but I looked at my GitHub repo and it actually shows all the builds that vercel does, and it seems that all failed in the last 18 days (code errors regarding putting getStaticPaths, getStaticProps and such), so the problem is mine!
Thank you!

**Reply by amyegan** [reply]
Oh no! Hopefully it isn't too troublesome to solve. I'm always happy to help out if you decide that you need another set of eyes on it.
